atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -2001,7 +2008,8 @@ private void commitWrites(TransactionService transactionService) {
             // this transaction before making progress.
             traced(
                     "postSweepEnqueueLockCheck",
-                    () -> preCommitRequirementValidator.throwIfImmutableTsOrCommitLocksExpired(commitLocksToken));
+                    () -> preCommitRequirementValidator.throwIfImmutableTsOrCommitLocksExpired(
```

Comment:
Checked that `preCommitRequirementValidator.throwIfImmutableTsOrCommitLocksExpired` also validates the immutableTs lock is valid, so no change in behavior.